### ISSUE TITLE
Unexpected EOF - syntax error

### ISSUE THREAD

**Issue by RalphMichel** [amiaopensource/vrecord#836]
After updating from version v2024-07-29 to v2024-08-28 I get the following error message after entering `vrecord -e`:
```
/opt/homebrew/bin/vrecord: line 772: unexpected EOF while looking for matching `)'
/opt/homebrew/bin/vrecord: line 2838: syntax error: unexpected end of file
```
I have already tried to delete and reinstall vrecord but the error persists. Have I possibly done something wrong or is the fault with brew?

I use it under macOS Sonoma 14.6.

**Comment by privatezero** [comment]
Hi @RalphMichel, I haven't been able to reproduce this error on my end so hard to say yet what exactly is going wrong. If you try running vrecord from a different source (such as from a [download of the repository](https://github.com/amiaopensource/vrecord/archive/refs/heads/main.zip)) do you get the same error?

**Comment by RalphMichel** [comment]
@privatezero: Yes, the same error message also appears when I run vrecord directly from the repository download. In the meantime, I have also updated vrecord on other computers and everything works fine. So the error is probably not with vrecord.

**Comment by RalphMichel** [comment]
The error was fixed with v2025-04-21. Thank you!
